www/gatsby-config.js

```diff
@@ -296,7 +296,7 @@ module.exports = {
                   limit: 10,
                   filter: {
                     frontmatter: { draft: { ne: true } }
-                    fileAbsolutePath: { regex: "/docs.blog/" }
+                    fields: { section: { eq: "blog" } }
```

Comment:
The other queries use ` released: { eq: true },` and not the `!draft`
I would expect that the feed is equal to the things on page...